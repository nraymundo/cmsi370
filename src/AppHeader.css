.AppHeader h1 {
  color: white;
  font-family: "Oswald", sans-serif;
  font-weight: 700;
  font-size: 50px;
}

.AppHeader {
  text-align: center;
}
