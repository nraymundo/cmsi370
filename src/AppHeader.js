import React from "react";

import "./AppHeader.css";

// Note how functional components that donâ€™t have additional logic simply return their markup.
const AppHeader = () => (
  <div className="AppHeader">
    <h1>Take That For Data</h1>
    <h2>Premier League | Bundesliga | Serie A | La Liga | Ligue 1</h2>
  </div>
);

export default AppHeader;
