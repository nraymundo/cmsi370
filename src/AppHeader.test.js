import React from "react";
import TestRenderer from "react-test-renderer";

import AppHeader from "./AppHeader.js";

it("should render the App Header", () => {
  const component = TestRenderer.create(<AppHeader />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
