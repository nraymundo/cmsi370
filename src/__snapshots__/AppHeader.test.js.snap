// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the App Header 1`] = `
<div
  className="AppHeader"
>
  <h1>
    Take That For Data
  </h1>
  <h2>
    Data from the teams in the Premier League, Bundesliga, Serie A, and La Liga
  </h2>
</div>
`;
